docker-compose.yml

```diff
@@ -12,8 +12,8 @@ services:
             - MEDIA_DATABASE_HOST=media-postgresql
             - MEDIA_DATABASE_PORT=5432
             - MEDIA_DATABASE_NAME=openslides
-            - MEDIA_DATABASE_USER=openslides
-            - MEDIA_DATABASE_PASSWORD=openslides
+            - MEDIA_DATABASE_USER_FILE=/run/secrets/media_database_user
+            - MEDIA_DATABASE_PASSWORD_FILE=/run/secrets/media_database_password
```

Comment:
I suppose, that the file `docker-compose.yml` is for a prod setup. If so, then please read the secrets from the secrets part of a docker-compose file like it is in OpenSlides/openslides-datastore-service#156 the case.

Reply:
Use the secrets mechanisms for prod and tests setup.